Ex2/ex2.js

```diff
@@ -0,0 +1,17 @@
+function eliminaProprietate(obiect, proprietate) {
+    // Creăm o copie a obiectului numita obiectNou folosind spread operatorul (...obiect)
+    const obiectNou = { ...obiect };
+    
+    // Ștergem proprietatea specificată din noul obiect cu operatorul delete
+    delete obiectNou[proprietate];
+    
+    // Returnăm noul obiect fără proprietate
+    return obiectNou;
+}
+
+//Input: obiect = { a: 1, b: 2 }, proprietate = “b”
+const obiect = { a: 1, b: 2 };
+const proprietate = "b";
+const rezultatFunctie = eliminaProprietate(obiect, proprietate);
```

Comment:
```suggestion
const rezultatObiect = eliminaProprietate(obiect, proprietate);
```
(opt) sa se puna un nume mai descriptiv, ex: `obiectModificat` `obiectFaraProprietate`